include/seqan3/range/view/all.hpp

```diff
@@ -114,48 +114,97 @@
- * `seqan3::input_range_concept` and of course also `seqan3::view_concept`. Most views also preserve stronger
+ * **Return range guarantees:** All views adaptors that are not *sink-only* return a range that meets at least
+ * `seqan3::input_range_concept` and also `seqan3::view_concept`. Most views also preserve stronger
  * properties, e.g. `seqan3::random_access_range_concept`, but this depends on the view. Some views also add
- * proporties not present on the input range, e.g. the range returned by `ranges::view::take_exactly` meets
+ * properties not present on the input range, e.g. the range returned by `ranges::view::take_exactly` meets
  * `seqan3::sized_range_concept`, independent of whether this was met by the input range.
- *
- * **Input range's reference type:** The reference type is the type the elements of the input range are accessed by
+ *   * *preserved* in this context means that the returned range satisfies this concept if it is also satisfied by the
+ * underlying range.
+ *   * *lost* means that this concept is never satisfied by the returned range, independent of whether the underlying
+ * range supports it.
+ *   * *guaranteed* means that this concept is always satisfied by the returned range, independent of whether the underlying
+ * range supports it.
+ *
+ * **Underlying range's reference type:** The reference type is the type the elements of the underlying range are
+ * accessed by
  * (since dereferencing an iterator or calling operator[] returns the reference type). The reference type may or may
  * not actually contain a `&` (see below). For many SeqAn specific views additional concept requirements are defined
  * for the input range's reference type, e.g. seqan3::view::complement can only operate on ranges whose elements are
  * nucleotides (meet seqan3::nucleotide_concept). In some case the type may even be a specific type or the result
  * of a metafunction.
  *
- * **Return range's reference type:** Conversely certain views make guarantees on the concepts satisfied by the
+ * **Returned range's reference type:** Conversely certain views make guarantees on the concepts satisfied by the
  * return range's reference type or even always have a fixed type, e.g. seqan3::view::complement operates on
- * nucleotides and of course also returns nucleotides. However, and this is important to note, the reference type
- * of seqan3::view::complement has any actual `&` removed from the input ranges' reference type (if originally present).
+ * nucleotides and of course also returns nucleotides and "seqan3::reference_t<urng_t>" would imply that
+ * the reference type is the same. However, and this is important to note, the reference type
+ * of seqan3::view::complement has any actual `&` removed from the underlying ranges' reference type (if originally present),
+ * this goes hand-in-hand with seqan3::output_range_concept being lost → original elements cannot be written to through
```

Comment:
is `→` an allowed character? At least I was never allowed to use `ö` for instance.

Reply:
Yes, our guide on this has always stated:

> **utf8:** All files are expected to be UTF-8 encoded. Special characters are allowed in documentation and string literals, but not in regular code (function names etc) and file names.